code/game/gamemodes/game_mode.dm

```diff
@@ -290,9 +290,11 @@
 		if(!continuous_sanity_checked) //make sure we have antags to be checking in the first place
 			for(var/mob/Player in GLOB.mob_list)
 				if(Player.mind)
-					if(Player.mind.special_role || LAZYLEN(Player.mind.antag_datums))
-						continuous_sanity_checked = 1
-						return 0
+					if(Player.mind.special_role)

```

Comment:
```suggestion
					if(Player.mind?.special_role)

```
To remove a level of indentation if you can be bothered

Reply:
done